engine/src/flutter/third_party/web_locale_keymap/lib/web_locale_keymap/key_mappings.g.dart

```diff
@@ -91,8 +91,8 @@ bool _isAscii(int charCode) {
 
 /// Returns whether the `char` is a single character of a letter or a digit.
 bool isLetter(int charCode) {
-  return (charCode >= _kLowerA && charCode <= _kLowerZ)
-      || (charCode >= _kUpperA && charCode <= _kUpperZ);
+  return (charCode >= _kLowerA && charCode <= _kLowerZ) ||
```

Comment:
Are the format changes intentional? If so it'd be preferred to be submitted in a separate PR for the clarity of this PR.